test/fixtures/lodash-fp-direct/expected.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var _lodashCollectionMap = require('lodash/collection/map');
+
+var _lodashCollectionMap2 = _interopRequireDefault(_lodashCollectionMap);
+
+var _lodashMathAdd = require('lodash/math/add');
+
+var _lodashMathAdd2 = _interopRequireDefault(_lodashMathAdd);
+
+var _lodashArrayTake = require('lodash/array/take');
+
+var _lodashArrayTake2 = _interopRequireDefault(_lodashArrayTake);
+
+var _lodashCollectionReject = require('lodash/collection/reject');
+
+var _lodashCollectionReject2 = _interopRequireDefault(_lodashCollectionReject);
+
+var _lodashFpConvert = require('lodash-fp/convert');
+
+var _lodashFpConvert2 = _interopRequireDefault(_lodashFpConvert);
+
+var _lodashFp = (0, _lodashFpConvert2['default'])({
```

Comment:
It's something `babel` adds, I'm not entirely sure why

https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=import%20x%20from%20%27x%27%3B%0A%0Avar%20y%20%3D%20x%28{}%29